### ISSUE TITLE
PFX still does not import into macOS keychain

### ISSUE THREAD

**Issue by Lotusshaney** [chris2511/xca#474]
Following on from https://github.com/chris2511/xca/pull/389#issuecomment-1760408048 This is still broken in v2.5.0

I have tried both PBE-SHA1-3DES and AES-256-CBC for the PKCS12 algorithm.

Both times importing the pfx into a keychain fails with "Sorry, you entered an invalid password."

If I convert the pfx using OpenSSL 3 with the "legacy" option the resulting pfx will import using the password without problem into the keychain

openssl pkcs12 -in xca.pfx -out xca.pem
openssl pkcs12 -export -in xca.pem -out xca2.pfx -legacy

**Comment by Alyoscha** [comment]
Same problem here using xca 2.5.0 on Apple M2 Pro, MacOS 13.6.3. Importing the pkcs12 into a Cisco ASA results in error.
Exporting again using openssl "-legacy" option works fine.

**Comment by chris2511** [comment]
Fixed with https://github.com/chris2511/xca/commit/4cb7b206a676ab5d00ce9407bfc027f78e2f71c2 for the next release.
Verifications with the action build from this [action run](https://github.com/chris2511/xca/actions/runs/7746700519) are welcome.

**Comment by Lotusshaney** [comment]
Hi, Not sure on the difference between build (macos, 5.15.2) and build (macos, 6.5.3) apart from size, but both appear to fix the problem

**Comment by azur538** [comment]
PFX does not import into MACOS Keychain 
I have successfully installed the pfx file on Windows. I have saved the file on a USB stick and transfered it on my hard disk (I have a NEO MAC). I have the following message "impossible to decode the provided data". However, when I try to open the file directly from an email sent on MAC with the extension .p12, the message is different : "invalid password".
If I do it again, I get another message: "system root keychain cannot be modified". When I try to modify trust settings, nothing happens.
Anyone can help?
